<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>821</width>
      <height>631</height>
     </rect>
    </property>
    <widget class="QWidget" name="home">
     <widget class="QPushButton" name="pag_autore">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>130</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>aggiungi autore</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pag_conferenza">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>230</y>
        <width>161</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>aggiungi conferenza</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="aggiungiAutore">
     <widget class="QPushButton" name="push_autore">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>430</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>aggiungi</string>
      </property>
     </widget>
     <widget class="QListWidget" name="lista_autori">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>120</y>
        <width>451</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="back_home">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>550</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;Umpush&quot;;
background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(128, 128, 128, 255), stop:1 rgba(255, 255, 255, 255));
</string>
      </property>
      <property name="text">
       <string>home</string>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>251</width>
        <height>288</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="idAutore">
         <property name="text">
          <string>Id:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="edit_idAutore"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="nomeAutore">
         <property name="text">
          <string>Nome:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="edit_nomeAutore"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="cognomeAutore">
         <property name="text">
          <string>Cognome:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="edit_cognomeAutore"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="afferenza">
         <property name="text">
          <string>Afferenza:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QTextEdit" name="edit_afferenza"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="aggiungiConferenza">
     <widget class="QPushButton" name="back_home_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>550</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;Umpush&quot;;
background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(128, 128, 128, 255), stop:1 rgba(255, 255, 255, 255));</string>
      </property>
      <property name="text">
       <string>home</string>
      </property>
     </widget>
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>401</width>
        <height>171</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QCalendarWidget::SingleSelection</enum>
      </property>
      <property name="horizontalHeaderFormat">
       <enum>QCalendarWidget::ShortDayNames</enum>
      </property>
      <property name="verticalHeaderFormat">
       <enum>QCalendarWidget::NoVerticalHeader</enum>
      </property>
      <property name="navigationBarVisible">
       <bool>true</bool>
      </property>
      <property name="dateEditEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>425</x>
        <y>130</y>
        <width>381</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>309</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>acronimo:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>luogo:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEdit_3"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>numero di Partecipanti:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="spinBox"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>nome:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
